### PR TITLE
query and integrate changelog during fetching

### PR DESCRIPTION
**Pull request by chriku**

### CODE REVIEW

**Comment by chriku** [comment]
Depends on #96

File: jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt [hunk 1]
```diff
@@ -100,13 +100,30 @@ final class JiraSync(
...
+                    appendPathSegments("issue")
+                    appendPathSegments(issueId)
+                    appendPathSegments("changelog")
+                    parameters.append("startAt", "$startAt")
+                }.second.body<ValueChangeLogContainer>()
+                issueCommentList.values.forEach {
+                    issueDataService.insertChangelogEntry(imsProject, issueId, it)
+                }
+                startAt = issueCommentList.startAt + issueCommentList.values.size
+                if (startAt >= issueCommentList.total) break
```
Comment (nk-coding) [inline jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt:116]: Brackets

Comment (chriku) [reply jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt]: 1bc373190e3a64750877f6f1630f89afd6902635

**Comment by chriku** [comment]
#108 has a different changelog integrator which would be the correct one for jira server and maybe also jira data center

so this one has to be easily disableable for data center usage

File: jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt [hunk 2]
```diff
@@ -102,20 +102,41 @@ final class JiraSync(
         issueList: MutableList<String>, imsProject: IMSProject
     ) {
         logger.info("ISSUE LIST $issueList")
+        for (issueId in issueList) {
```
Comment (nk-coding) [inline jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt:105]: could this be split into two methods?
an automatic refactoring should be capable of doing this

Comment (chriku) [reply jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt:105]: as we still have to break through multiple levels, this would make it unreadable

Comment (chriku) [reply jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt:104]: 7a5372a9ec59b96bff03e731bbbf8ecf4a7c7214

Comment (nk-coding) [reply jira/src/main/kotlin/gropius/sync/jira/JiraSync.kt:104]: Thx

**Review by nk-coding — APPROVED** [review]

**Pull request merged**